apps/right-to-rent-check/translations/src/en/fields.json

```diff
@@ -1,5 +1,33 @@
 {
-  "name": {
-    "label": "Name"
+  "living-status": {
+    "options": {
+      "yes": {
+        "label": "Yes"
+      },
+      "no": {
+        "label": "No"
+      }
+    }
+  },
+  "tenant-name": {
+    "label": "Prospective tenant's full name"
+  },
+  "tenant-dob": {
+    "legend": "Date of Birth",
+    "hint": "For example, 12 3 1980",
+    "parts": {
+      "day": {
+        "label": "Day"
+      },
+      "month": {
+        "label": "Month"
+      },
+      "year": {
+        "label": "Year"
+      }
+    }
```

Comment:
you don't need parts - these are the defaults. Only use parts if you want to use something other than Day, Month Year